test/test_distributions.py

```diff
@@ -101,6 +87,30 @@
         {'rate': Variable(torch.randn(5, 5).abs(), requires_grad=True)},
         {'rate': Variable(torch.randn(1).abs(), requires_grad=True)},
     ]),
+    Example(Gamma, [
+        {
+            'alpha': Variable(torch.exp(torch.randn(2, 3)), requires_grad=True),
+            'beta': Variable(torch.exp(torch.randn(2, 3)), requires_grad=True),
+        },
+        {
+            'alpha': Variable(torch.exp(torch.randn(1)), requires_grad=True),
+            'beta': Variable(torch.exp(torch.randn(1)), requires_grad=True),
+        },
+    ]),
+    Example(Laplace, [
```

Comment:
FYI while it's nice to keep these in alphabetical order, by moving these around you will cause merge conflicts for all the other PRs that are currently under review.